content/doc/index.adoc

```diff
@@ -4,26 +4,45 @@ title: Jenkins Documentation
 section: doc
 ---
 
+== What is Jenkins?
+
 Jenkins is a self-contained, open source automation server which can be used to
 automate all sorts of tasks such as building, testing, and deploying software.
+
 Jenkins can be installed through native system packages, Docker, or even run
-standalone by any machine with the Java Runtime Environment installed.
+standalone by any machine with the Java Runtime Environment (JRE) installed.
 
 == Guided Tour
 
-This guided tour will use the "standalone" Jenkins distribution which requires
-Java 8. A system with more than 512MB of RAM is also recommended.
+This guided tour introduces you to the basics of using Jenkins and its main
+feature, Jenkins Pipeline. This tour uses the \'standalone' Jenkins distribution,
+which runs locally on your own machine.
+
+=== Prerequisites
+
+* Java 8 (either a JRE or Java Development Kit (JDK) is fine)
+* https://www.docker.com/[Docker] (navigate to the \'Get Docker' link at the top
+  of the page to access the Docker download that's suitable for your platform)
+* A machine with:
+** 256 MB of RAM, although more than 512MB is recommended
+** 10 GB of drive space (for Jenkins and your Docker image)
+
+=== Download and run Jenkins
 
 . http://mirrors.jenkins.io/war-stable/latest/jenkins.war[Download Jenkins].
-. Open up a terminal in the download directory and run `java -jar jenkins.war --httpPort=8080`
-. Browse to `http://localhost:8080` and follow the instructions to complete the installation.
+. Open up a terminal in the download directory.
+. Run `java -jar jenkins.war --httpPort=8080`.
+. Browse to `http://localhost:8080`.
+. Follow the instructions to complete the installation.
 . Many Pipeline examples require an
   https://docs.docker.com/engine/installation[installed Docker]
   on the same computer as Jenkins.
 
 When the installation is complete, start putting Jenkins to work and create a
 link:book/pipeline[Pipeline].
 
+=== What is a Jenkins Pipeline?
```

Comment:
Maybe this section belongs at the top of "Create your first Pipeline" https://jenkins.io/doc/pipeline/tour/hello-world/ ?   Might be out of scope for this PR.

Reply:
I agree that this segment doesn't belong here and I'll deal with this in a separate PR.

Putting this segment at the top of the next "Creating your first Pipeline" page would be more appropriate, but since this section of the documentation (i.e. Guided Tour) is more like a set of mini tutorials, perhaps we should consider "merging" this "What is a Jenkins Pipeline?" segment into the introductory material of the [Pipeline section](https://jenkins.io/doc/book/pipeline/) of the User Handbook? (This'll entail a little re-writing.)

By doing this, we could:

- Link to the description/definition for what a Jenkins Pipeline is in the User Handbook (from the Guided Tour) and
- Remove partially duplicated content in this documentation.

FYI - For consistency with the other pages, I changed 'Create...' to 'Creating...' in the title of the 2nd Guided Tour page.

Reply:
@gilesgas - Sounds good.  I added this comment to WEBSITE-399, since it will probably need to be addressed at the same time that happens.

Reply:
Thanks for that @bitwiseman .